compiler/packages/babel-plugin-react-compiler/src/Inference/InferEffectDependencies.ts

```diff
@@ -97,6 +97,17 @@ export function inferEffectDependencies(fn: HIRFunction): void {
             autodepFnLoads.set(lvalue.identifier.id, numRequiredArgs);
           }
         }
+      } else if (
+        value.kind === 'LoadGlobal' &&
+        value.binding.kind === 'ImportDefault'
+      ) {
+        const moduleTargets = autodepFnConfigs.get(value.binding.module);
+        if (moduleTargets != null) {
+          const numRequiredArgs = moduleTargets.get('default');
```

Comment:
Does it make sense to make `default` a constant in `Environment` for grep-ability?

Reply:
yea, sounds reasonable to me! I like grep-ability + find-refs-ability